<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="4">
     <widget class="QPushButton" name="CreateFuel_button">
      <property name="text">
       <string>Create Fuel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QLabel" name="State_label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No oil in station</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="Money_label">
        <property name="text">
         <string>Money: 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Oil_label">
        <property name="text">
         <string>Oil: 10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Fuel_label">
        <property name="text">
         <string>Fuel: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QSlider" name="horizontalSlider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Return_button">
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Insert_button">
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
